src/status_im/ui/screens/signing/views.cljs

```diff
@@ -0,0 +1,197 @@
+(ns status-im.ui.screens.signing.views
+  (:require-macros [status-im.utils.views :as views])
+  (:require [status-im.ui.components.react :as react]
+            [re-frame.core :as re-frame]
+            [status-im.ui.components.colors :as colors]
+            [status-im.ui.components.animation :as anim]
+            [reagent.core :as reagent]
+            [status-im.ui.components.list-item.views :as list-item]
+            [status-im.ui.components.button.view :as button]
+            [status-im.ui.screens.wallet.utils :as wallet.utils]
+            [status-im.ui.components.list.views :as list]
+            [status-im.ui.components.chat-icon.screen :as chat-icon]
+            [status-im.ui.components.icons.vector-icons :as icons]
+            [status-im.ui.components.text-input.view :as text-input]
+            [status-im.i18n :as i18n]
+            [status-im.utils.security :as security]
+            [status-im.ui.screens.signing.sheets :as sheets]
+            [status-im.ethereum.tokens :as tokens]
+            [clojure.string :as string]
+            [status-im.ui.screens.signing.styles :as styles]))
+
+(defn hide-panel-anim
+  [bottom-anim-value alpha-value window-height]
```

Comment:
could you use a translateY value here instead of height in order to use nativeDrivers?

Reply:
(you can't do height because native drivers don't support it but translation is similar)